source/modules/MEI.shared.xml

```diff
@@ -3469,6 +3469,38 @@
         C to C♯ and one from C to D♭.</p>
     </remarks>
   </classSpec>
+  <classSpec ident="att.tuning" module="MEI.shared" type="atts">
+    <desc>Attributes that describe tuning.</desc>
+    <attList>
+      <attDef ident="tune.accid" usage="opt">
+        <desc>Holds a semitone shift to a tuning reference pitch.</desc>
+        <datatype>
+          <rng:ref name="data.ACCIDENTAL.GESTURAL"/>
+        </datatype>
+      </attDef>
+      <attDef ident="tune.Hz" usage="opt">
+        <desc>Holds a value for cycles per second, <abbr>i.e.</abbr>, Hertz, for a tuning reference pitch.</desc>
+        <datatype>
+          <rng:data type="decimal"/>
+        </datatype>
+      </attDef>
+      <attDef ident="tune.pname" usage="opt">
+        <desc>Holds the pitch name of a tuning reference pitch.</desc>
+        <datatype>
+          <rng:ref name="data.PITCHNAME"/>
+        </datatype>
+      </attDef>
+      <attDef ident="tune.temper" usage="opt">
+        <desc>Provides an indication of the tuning system, <val>just</val>, for example.</desc>
+        <datatype>
+          <rng:ref name="data.TEMPERAMENT"/>
+        </datatype>
+      </attDef>
+    </attList>
+    <remarks>
+      <p>If tune.pname is not specified, a4 can be assumed as the reference pitch.</p>
```

Comment:
Is this supposed to say "A 440"? Not sure if "a4" means pname + oct?

Reply:
It's referring to scientific pitch notation. "A440" would mean pitch + tuning.

Reply:
Sure; but does the "4" mean the octave then? If so, how can we capture the octave in the tuning attributes? As far as I can tell there is no way of specifying that the tuning is, for example, `c5` (to make up an example).

Reply:
Yes. So either we add another `tune.oct` attribute, or we restrict tunings to the "fourth" octave.

Reply:
Do you have some real-world examples of where this might be used, or what experiences you've had that prompted the change? I'm struggling to think of some, so having some examples would help.

Specific questions:

 - In this case, I'm not sure what `tune.pname` means; My assumption is that it's necessary to have a fixed Hz reference point for the note name, e.g., A=440; A=466, etc. So just having `tune.pname="A"` seems to omit half of the necessary information?
 - Are there cases where the reference pitch would be anything other than A? I've never heard A=440 referred to as Bb=466... And in that specific case they would be equivalent, so it would be unnecessarily complicated. Can't you always reference a `pname` to A?
 - And, in the case that the reference pitch *is* something different, wouldn't you also need the temperament as well? A=466 or 440 doesn't tell you the temperament, so you need to include that to know the distance to the next note -- A=466 with meantone temperament means something, but what would `tune.pname=Bb`, `tune.temper=meantone` mean? I would guess you would need the fixed `tune.Hz` for this -- `tune.pname=Bb  tune.Hz="???" tune.temper="meantone"`

Reply:
ODD meeting (July 2022): We agreed that the meaning and description of tune.pname is misleading and needs clarification. This should be done in another PR, since the PR here does not change any existing attributes, but just moves tuning related things into a separate class. @ahankinson Are you ok with resolving this conversation?

Reply:
outdated